packages/client/src/app/components/comments/comments.component.ts

```diff
@@ -157,8 +158,17 @@ export class CommentsComponent implements OnInit {
       return;
     }
 
+    const contentKind: ContentKind = 
+      this.itemKind === ItemKind.Blog ? ContentKind.BlogContent
+      : this.itemKind === ItemKind.BlogContent ? ContentKind.BlogContent
+      : this.itemKind === ItemKind.Work ? ContentKind.WorkContent
+      : this.itemKind === ItemKind.WorkContent ? ContentKind.WorkContent
+      : this.itemKind === ItemKind.NewsContent ? ContentKind.NewsContent
+      : ContentKind.WorkContent;
```

Comment:
I hacked this in kind of haphazardly, just so I could test comments. Feel free to yell at me to fix this.